bots/tests-invoke

```diff
@@ -114,7 +114,7 @@ class PullTask(object):
 
         # build a unique file name for this test run
         id_context = self.context
-        if self.test_project:  # disambiguate when running several external projects
+        if self.test_project != api.repo :  # disambiguate when running several external projects
```

Comment:
This is just for the log name, right? I think this special case could go, it doesn't matter if the log name contains the cockpit project.